doc_source/falco-fargate-ptrace-security.md

```diff
@@ -0,0 +1,119 @@
+# Securing Fargate Tasks with Falco Runtime Security
+
+ECS Fargate `1.4.0` [announced support](https://aws.amazon.com/about-aws/whats-new/2020/04/aws-fargate-launches-platform-version-14/) for the `CAP_SYS_PTRACE` linux capability.
+
+This new feature can be implemented with [Falco](falco.org), an open source runtime security project originally built by Sysdig, Inc and later donated to the Cloud Native Computing Foundation.
+
+Falco uses `ptrace(2)` with Fargate in ECS to detect anomalous behavior at runtime. 
+
+Falco can be configured to send alerts to STDOUT such that ECS can consume the logs. They can then be used to trigger alerts and alarms using AWS logging solutions such as CloudWatch. 
+
+## Dependencies
+
+In order to begin securing a task in ECS a few components need to be installed within the container.
+
+ - Falco userspace daemon
+ - Falco pdig tracing utility
+ 
+After these components are installed in the container, Falco can be used in the following ways.
+
+ - Launch an arbitrary process, and begin tracing the original process and all of it's child processes. 
+ - Attach to a running process, and begin tracing the process and all of it's child processes. 
+ 
+In order for the Falco `pdig` utility to work, the `CAP_SYS_PTRACE` capability must be enabled in a task definition for a Fargate 1.4 or greater ECS Cluster. 
+
+Add the following section to the container JSON you wish to secure while creating the task definition. 
+
+```json
+            "linuxParameters": {
+                "capabilities": {
+                    "add": [
+                        "SYS_PTRACE"
+                    ],
+                    "drop": null
+                }
+            },
+```
+
+## Installing Falco
+
+_Note:_ The Falco `pdig` components are targetted to be released in an upcoming Falco release. This documentation will need to be updated after the initial release.
+
+For now you can use the `krisnova/falco-trace:latest` container image which is built on debian and contains Falco and `pdig` pre-installed. 
+The original work to get this working can be found at [github.com/kris-nova/falco-trace](https://github.com/kris-nova/falcotrace).
```

Comment:
These lines (40-44) will ultimately be removed before I remove the WIP flag from the PR.

But until the Falco community is able to cut a release with these updates, this is a place for folks to kick the tires.